recipe/session/testingUtils.go

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
Why can't all of these files be moved to tests folder and these functions exposed from that package?

Reply:
It is because of the function called resetAll if you want to expose resetAll from inside the unittesting package then you need to import all the recipes inside that package that will cause a cyclical import because all recipes need the unittesting package aswell

Reply:
cool